src/app/js/test.spec.js

```diff
@@ -1,2 +1,7 @@
+import Enzyme from 'enzyme';
```

Comment:
This should not be in a `.spec.js` file, which already matches mocha's test files pattern